### PR TITLE
Add quote

### PR DESCRIPTION
**Pull request by imanishbarnwal**
## ✅ Checklist
- [x] The quote(s) added are not ❌ duplicates.
- [x] The sentence attribute does not have inner quotation marks and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

**Fixed Issue:** #13

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -1,4 +1,32 @@
 [
+  {
```
Comment (jayehernandez) [inline server/data/quotes.json:2]: Hi @imanishbarnwal, the quotes should be about **learning foreign languages**.

Updated the checklist just so it's clear. Hope you can change the quotes!

Comment (imanishbarnwal) [reply server/data/quotes.json:2]: Ohh okay sure thanks will do it

Comment (imanishbarnwal) [reply server/data/quotes.json:2]: Now check them I just updated all of it.

Comment (jayehernandez) [reply server/data/quotes.json:2]: Thanks looks fine now!

**Comment by jayehernandez** [comment]
Thanks for contributing @imanishbarnwal!

@all-contributors please add @imanishbarnwal for code

**Comment by jayehernandez** [comment]
@all-contributors please add @imanishbarnwal for code

**Pull request merged**